README.md

```diff
@@ -467,6 +467,8 @@ Available options (all `false` by default):
 
 + `async`: Asynchronous execution. Defaults to true if a callback is provided.
 + `silent`: Do not echo program output to console.
++ `cwd`: Working directory. Defaults to process.cwd().
```

Comment:
Make this change in the comments. Run `scripts/generate-docs.js` to generate the README